css/app.css

```diff
@@ -8,7 +16,10 @@ body {
 *,
 *:before,
 *:after {
-    box-sizing: inherit;
+  -webkit-box-sizing: border-box;
```

Comment:
Don't use vendor-specific CSS. We will use a transpiler to add all that automatically for us at release time.